src/Main.java

```diff
@@ -0,0 +1,400 @@
+
+        //Player Stats
+        player.attack = 5;
+        player.health = 20;
+        player.stamina = 20;
+        int h = 1;
+        int i = 1;
+
+        //Zombie Stats
+        zombie.name = "Zombie";
+        zombie.attack = 2;
+        zombie.health = 10;
+
+        //Ghost Stats
+        ghost.name = "Ghost";
+        ghost.attack = 2;
+        ghost.health = 10;
+
+        //Goblin Stats
+        goblin.name = "Goblin";
+        goblin.attack = 2;
+        goblin.health = 10;
+
+        //Takes the combined stats of all monsters killed and fights the player
+        //With these stats he's gonna be op so there are going to be dodge frames
+        //To fight him you have to dodge his attacks by pressing certain keys at certain times
+        int monstersKilled = 0;
+        int combinedAttack = 0;
+        int combinedHealth = 0;
+        if (monstersKilled == 10) {
+            System.out.println("...");
+            Thread.sleep(3000);
+            System.out.println("A new enemy appeared...");
+            Thread.sleep(3000);
+            System.out.println("T H E  D U N G E O N  B O S S");
+            Thread.sleep(2000);
+            System.out.println("Health: " + dungeonBoss.health);
+            Thread.sleep(1000);
+            System.out.println("Attack: " + dungeonBoss.attack);
+
+        }
+        dungeonBoss.name = "Dungeon Boss";
+        dungeonBoss.attack = combinedAttack;
+        dungeonBoss.health = combinedHealth;
+
+
+
+
+
+       System.out.println("Welcome to dungeon run!");
+       Thread.sleep(2000);
+       System.out.println("In this game you will have to slay as many monsters as possible and stay alive");
+       Thread.sleep(3000);
+       System.out.println("What would you like you character's name to be?");
+       String userName = playerInput.nextLine();
+        player.name = userName;
+        System.out.println("Great Choice!");
+        Thread.sleep(2000);
+        System.out.println("Here are " + userName + "'s Stats: ");
+        Thread.sleep(1000);
+        System.out.println("Attack: " + player.attack);
+        Thread.sleep(1000);
+        System.out.println("Health: " + player.health);
+        Thread.sleep(1000);
+        System.out.println("Stamina: " + player.stamina);
+        Thread.sleep(1000);
+        System.out.println("(Besides your attack which gets capped at 10 your base stats cannot go over 20)");
+        Thread.sleep(4000);
+       System.out.println("Lets begin!");
+        Thread.sleep(2000);
+
+
+       System.out.println(userName + "'s first encounter is a " + randomMonster + "!");
+       Character monster;
+       if (randomMonster.equals ("Zombie")) {
+           monster = zombie;
+           Thread.sleep(2000);
+           System.out.println("Zombie Stats: ");
+           Thread.sleep(1000);
+           System.out.println("Attack: " + zombie.attack);
+           Thread.sleep(1000);
+           System.out.println("Health: " + zombie.health);
```

Comment:
Consider wrapping this, lines 129-135, and lines 138-144 in a function. In fact, I think you only need to create one function for all three of these sections. 

Look into function parameters, they may help!